src/pages/workspaces/workspace/analysis/file-utils.test.ts

```diff
@@ -109,5 +121,113 @@ describe('file-utils', () => {
       // Assert
       expect(calledMock).toHaveBeenCalledWith(defaultAzureWorkspace.workspace.workspaceId)
     })
+
+    it('creates a file with a GCP workspace', async () => {
+      // Arrange
+      const fileList: AnalysisFile[] = [getTestFile('test/file1.ipynb' as AbsolutePath), getTestFile('test/file2.ipynb' as AbsolutePath)]
+
+      const listAnalyses = jest.fn(() => Promise.resolve(fileList))
+      const create = jest.fn(() => Promise.resolve())
+      const analysisMock: Partial<GoogleStorageContract['analysis']> = jest.fn(() => ({
+        create
+      }))
+      const googleStorageMock: Partial<GoogleStorageContract> = ({
+        listAnalyses,
+        analysis: analysisMock as GoogleStorageContract['analysis']
+      })
+      asMockedFn(GoogleStorage).mockImplementation(() => googleStorageMock as GoogleStorageContract)
+
+      // Act
+      workspaceStore.set(defaultGoogleWorkspace)
+      const { result: hookReturnRef, waitForNextUpdate } = renderHook(() => useAnalysisFiles())
+      await waitForNextUpdate()
+      await act(async () => {
+        await hookReturnRef.current.create('AnalysisFile', toolLabels.Jupyter, 'myContents')
+      })
```

Comment:
Not necessary to change, but one line `act`s can be written more concisely as:

```suggestion
      await act(() => hookReturnRef.current.create('AnalysisFile', toolLabels.Jupyter, 'myContents'))
```